R/milestone.R

```diff
@@ -23,13 +23,18 @@
 #'   - `event` - Event indicator.
 #'   - `treatment` - Grouping variable.
 #' @param ms_time Milestone analysis time.
-#'
+#' @param test_type Method to build the test statistics.
+#' There are 2 options:
+#'   - `"native"`: a native approach by dividing the KM survival difference by its standard derivations,
+#'   see equation (1) of Klein, J. P., Logan, B., Harhoff, M., & Andersen, P. K. (2007).
+#'   - `"log-log"`: a log-log transformation of the survival, see equation (3) of
+#'   Klein, J. P., Logan, B., Harhoff, M., & Andersen, P. K. (2007).
 #' @return A list frame containing:
-#' - `method` - The method, always `"milestone"`.
-#' - `parameter` - Milestone time point.
-#' - `estimation` - Survival difference between the experimental and control arm.
-#' - `se` - Standard error of the control and experimental arm.
-#' - `z` - Test statistics.
+#'   - `method` - The method, always `"milestone"`.
+#'   - `parameter` - Milestone time point.
+#'   - `estimation` - Survival difference between the experimental and control arm.
+#'   - `se` - Standard error of the control and experimental arm.
```

Comment:
The `se` result was changed by this PR, but the documentation did not change. Previously it returned two standard error values (from `fit_res$std.err`), but now it only returns a single standard error. Is is the standard error of the control arm, experimental arm, or some combination? We should update the documentation for this return value

Reply:
Agreed, this should be a quick fix for @LittleBeannie